hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestNSSummaryEndpoint.java

```diff
@@ -51,16 +57,13 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.List;
+import java.util.*;
```

Comment:
Let's change this back.

You can disable intellij's wildcard imports feature: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcards-for-class